src/models/hooks/eventReportPilot.js

```diff
@@ -98,6 +100,76 @@ const notifyVisionAndGoalComplete = async (_sequelize, instance) => {
   }
 };
 
+/**
+ * This hook updates `Goal.name` for all goals that are associated with this training report.
+ */
+const updateGoalText = async (sequelize, instance, options) => {
```

Comment:
should this create a goal if one doesn't exist? given the majority of our events will already exist when this is deployed

Reply:
majority of our sessions, I mean

Reply:
1336c303bcf44aee6425e7da06252deb64f859d3